uaclient/entitlements/fips.py

```diff
@@ -93,8 +102,23 @@ def messaging(
         self
     ) -> "Dict[str, List[Union[str, Tuple[Callable, Dict]]]]":
         return {
-            "post_enable": [
-                "FIPS Updates configured and pending, please reboot to make"
-                " active."
-            ]
+            "pre_enable": [
+                (
+                    util.prompt_for_confirmation,
+                    {
+                        "msg": status.PROMPT_FIPS_UPDATES_PRE_ENABLE,
+                        "assume_yes": self.assume_yes,
+                    },
+                )
+            ],
+            "post_enable": [status.MESSAGE_ENABLE_REBOOT_REQUIRED],
+            "pre_disable": [
```

Comment:
so post-disable will be future coming?

Reply:
@mitechie correct. post_disable will also have to handle the case where FIPS doesn't uninstall packages (which is unique to UA client defaults which attempt to remove any packages on disable)